src/calculateRentalCost.js

```diff
@@ -29,6 +29,24 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const cost = 40;
+  const total = cost * days;
+  const largeDiscount = 50;
+  const smallDiscount = 20;
+  const largeDiscountRequirement = 7;
+  const smallDiscountRequirement = 3;
+
+  if (days <= 2) {
+    return total;
+  }
+
+  if (days < largeDiscountRequirement) {
+    return total - smallDiscount;
+  }
+
+  if (days >= smallDiscountRequirement) {
+    return total - largeDiscount;
+  }
 }
```

Comment:
Almost! But try to turn them around. 
Start with this one 
```
if (days >= smallDiscountRequirement) {
    return total - largeDiscount;
  }
  ```
  and use max 2 if statements